### PR TITLE
Updates to support rails 5.2

### PR DESCRIPTION
**Pull request by zbelzer**
The uses of ActiveRecord and ActiveSupport are very minor and don't appear to contain any breaking changes.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,21 +1,20 @@
 source "https://rubygems.org"
 
+ruby "2.4.6"
+
 # Specify your gem's dependencies in flat_map.gemspec
 gemspec
 
-group :development do
```
Comment (zbelzer) [inline Gemfile]: Groups are meaningless in vanilla gem environments.

File: HornsAndHooves-flat_map.gemspec [hunk 2]
```diff
@@ -19,11 +19,11 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   # specify any dependencies here; for example:
-  s.add_dependency(%q<activesupport>, ["~> 4.2"])
-  s.add_dependency(%q<activerecord>, ["~> 4.2"])
-  s.add_dependency(%q<yard>, [">= 0"])
+  s.add_dependency "activesupport", "> 4.2", "< 6.0.0"
```
Comment (albertosaurus) [inline HornsAndHooves-flat_map.gemspec]: Why can't this be `>= 4.2`?

Comment (zbelzer) [reply HornsAndHooves-flat_map.gemspec]: That's what I started with and I chickened out with the rationale that I didn't test it with 6.

Comment (albertosaurus) [reply HornsAndHooves-flat_map.gemspec]: No, I mean `">= 4.2", "< 6.0.0"`

**Comment by zbelzer** [comment]
Oh

**Pull request merged**